code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -95,6 +99,7 @@
 	drink_icon = "absinthebottle"
 	drink_name = "Glass of Absinthe"
 	drink_desc = "The green fairy is going to get you now!"
+	taste_message = "fucking pain"
```

Comment:
This should say 'tastes like aniseed and pain' or similar. I quite like absinthe.